src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collection-view/main.js

```diff
@@ -316,6 +316,7 @@ define([
                 {
                     el: this.$find(constants.datagridSelector),
                     instanceName: this.options.instanceName,
+                    storageName: this.options.instanceName,
```

Comment:
This doesn't seem to be enough. Since the id is not included in the instance name, the search parameter will be the same for all collections. Is this the desired behavior? Because it felt strange to me while clicking through it.

However, I think it would also feel strange to have a different search for every collection. Maybe not storing it at all is an option here?

@chirimoya Can you please also add your opinion here?